### ISSUE TITLE
coding issues

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-evaluation-tools-list#108]
From Brian in the survey:
> There are some minor accessibility issues with the code but not serious enough to hold back publishing. Another evaluation should be done to fix these minor issues...

We want to be a model for good accessibility, so it would be best if they are found and fixed before publication.

@mhansma96

**Comment by MicheleAWilliams-A11y** [comment]
I found some missing form field labels when running WAVE so I think this is high. WAVE doesn't let you export the report (at least not the free version) but hopefully the team can use it (or equivalents like Accessibility Insights) to do a check before publishing.

**Comment by beltonTPG** [comment]
@iadawn the first issue I found was with the filter disclosure widgets and the tool tips within them.
The existing code:
```
<fieldset id="purpose" collapsed="false" class="default">
    <legend class="label collapsible" tabindex="0">Purpose
        <abbr title="Type of evaluation a tool supports" class="toggletip-container">
            <img alt="Type of evaluation a tool supports" data-toggletip-content="Type of evaluation a tool supports" tabindex="0" src="/content-images/wai-evaluation-tools-list/info.png">
            <span class="toggletip-span" role="status"></span>
        </abbr>
        <svg focusable="false" aria-hidden="true" class="icon-chevron-up "><use xlink:href="/assets/images/icons.svg#icon-chevron-up"></use></svg>
    </legend>
    <div class="options">
        ...
    </div>
</fieldset>
```

- The legend has a tabindex="0" but a legend element is not an interactive element. there is also no state information for the disclosure widget (aria-expanded).
- The legend (and therefore the fieldset) also has an accessible name of "Purpose Type of evaluation a tool supports" which is a combination of the legend text and the alt of the image.
- The img element (which is the trigger for a tool tip) is also an non-interactive element that has a tabindex="0", so we have nested interactive elements, which we shouldn't have.

The tool tip should be outside of the legend, so that a) it doesn't factor into the accName of the legend element, and b) we don't have nested interactive elements.
